app/user_filters/models.py

```diff
@@ -0,0 +1,13 @@
+import uuid as uuidlib
+
+from django.contrib.auth.models import User
+from django.db import models
+from djsonb import fields as jfields
+
+
+class SavedFilter(models.Model):
+    """A filter saved by the user for later reference"""
+    uuid = models.UUIDField(primary_key=True, default=uuidlib.uuid4, editable=False)
+    label = models.CharField(max_length=255)
+    owner = models.ForeignKey(User)
+    filter = jfields.JsonField()
```

Comment:
Oh good call; I was considering other names for this anyway, so I'll make the change.